test/integration/targets/vmware_content_library/content_lib_details.yml

```diff
@@ -0,0 +1,34 @@
+# Test code for the vmware_content_library_info.
+# Copyright: (c) 2019, Pavan Bidkar <pbidkar@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- when: vcsim is not defined
+  block:
+
+    # Get List of Content Libraries
+    - name: Get List of Content Libraries
+      vmware_content_library_info:
+        hostname: '{{ vcenter_hostname }}'
+        username: '{{ vcenter_username }}'
+        password: '{{ vcenter_password }}'
+        validate_certs: False
+      register: content_library_info
+
+    - set_fact: content_libraries="{{ content_library_info['content_libs'] }}"
+
+    - set_fact: content_library="{{ content_library_info['content_libs'][0] }}"
+
+    # Get Details of content library
+    - name: Get facts about Content Library
+      vmware_tag_facts:
```

Comment:
I think this task should be changed from `vmware_tag_facts` to `vmware_content_library_info`

Reply:
Oops! sorry for such misses. I should have copied my working playbook instead of Tagging test .ymls
I will make sure to handle such typos before raising PR

Thanks